wasmedge-ggml-llama-interactive/src/main.rs

```diff
@@ -32,7 +33,10 @@ fn main() {
         _ => (),
     };
     match env::var("stream_stdout") {
-        Ok(val) => options["stream-stdout"] = serde_json::from_str(val.as_str()).unwrap(),
+        Ok(val) => {
+            options["stream-stdout"] = serde_json::from_str(val.as_str()).unwrap();
+            stream_stdout = true;
```

Comment:
Maybe `stream_stdout = options["stream-stdout"]` to support `stream_stdout=false` ?

Reply:
Fixed.